### PR TITLE
docs: cleanup contribution guide

### PR DESCRIPTION
**Pull request by cdtinney**
Cleanup the contribution guide -- grammar, spelling, clarity, structure, etc.

### CODE REVIEW

**Comment by juanpicado** [comment]
Thanks that was really need it :)

**Comment by cdtinney** [comment]
I've also realized there aren't any instructions for setting up a development environment!

Running `npm install` and `npm run dev:start` results in many errors. I will tackle that in another MR.

**Comment by juanpicado** [comment]
Yes there is in the wiki section

**Comment by juanpicado** [comment]
We put it here because is easier to update

**Comment by cdtinney** [comment]
That makes perfect sense. I've added some references to help others find the wiki from the contribution doc.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,59 +1,100 @@
...
+If so, up-vote it (using GitHub reactions) or add additional helpful details to
+the existing issue to show that it's affecting multiple people.
+ 
+### Check Website For Solution
+
+Some of the most popular topics can be found in our website(http://www.verdaccio.org/docs/en/installation.html)
+
+### Chat
+
+Questions can be asked in [gitter](https://gitter.im/verdaccio/)
```
Comment (juanpicado) [inline CONTRIBUTING.md]: Here we don't use **gitter** anymore, we have created a new easy link that redirect to Discord (http://chat.verdaccio.org/)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -62,63 +103,90 @@ to follow step 3 and beyond:
...
 
-At this point you have ready your changes, your new feature it's ready to be shipped, but, to avoid delays to merge, please be aware the build must past.
+Before committing, **you must ensure there are no linting errors and
+all tests pass.**
 
-Before commit, run the test command:
+To do this, run:
 
 ```bash
 yarn test
```
Comment (juanpicado) [inline CONTRIBUTING.md]: To be more acurate, we should use `yarn test:all` that run all test included the e2e, it takes a bit more time but are the ones are run on CircleCi.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -62,63 +103,90 @@ to follow step 3 and beyond:
...
-Plugins are Add-ons that extend the functionality of the application. Whether you want develop your own plugin I'd suggest do the following:
+If you want to develop your own plugin:
 
-1. Check whether there is a legacy sinopia plugin for the feature that you need at [npmjs](https://www.npmjs.com/search?q=sinopia).
-2. There is a [life-cycle to load a plugin](https://github.com/verdaccio/verdaccio/blob/master/lib/plugin-loader.js#L22) you should keep on mind.
-3. You are free to host your plugin in your repository, whether you want to host within in our organization, feel free to ask, we'll happy to host it.
-4. Try a describe widely your plugin to provide a deeply understanding to your users.
+1. Check whether there is a legacy Sinopia plugin for the feature that you need
+    via [npmjs](https://www.npmjs.com/search?q=sinopia)
+2. Keep in mind the [life-cycle to load a plugin](https://github.com/verdaccio/verdaccio/blob/master/lib/plugin-loader.js#L22)
```
Comment (juanpicado) [inline CONTRIBUTING.md]: This link is broken, I think we should point to https://verdaccio.org/docs/en/dev-plugins instead

**Comment by juanpicado** [comment]
I meant, I like much more in your PR than is is now in master :-)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,59 +1,100 @@
 # Contributing to Verdaccio
 
-We are happy you wish to contribute this project, for that reason we want to board you with this guide.
+We are happy that you wish to contribute this project. For that reason, we
```
Comment (ayusharma) [inline CONTRIBUTING.md]: `contribute this` --> `contribute to this`

**Pull request merged**